src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerLogsContainer.groovy

```diff
@@ -97,6 +97,11 @@ class DockerLogsContainer extends DockerExistingContainer {
     @Optional
     Writer sink
 
+    // Allows subclasses to carry their own logic
+    protected Date getInternalSince() {
+        return since.getOrNull();
```

Comment:
We can clean it up but let's remove the comma at the end.

Reply:
Ah, sorry, too much Java muscle memory 😉